### PR TITLE
Support for pod annotations

### PR DESCRIPTION
**Pull request by MaraisKruger**
## Description

Add the ability to add annotations to the pods of deployments and cronjobs

## Types of changes

_Put an `x` in the boxes that apply._

- [ ] Bugfix (`fix`): A non-breaking change which fixes an issue or bug
- [x] New Feature (`feat`): A non-breaking change which adds a new feature or new functionality
- [ ] Tests (`test`): Adding missing tests or correcting existing tests
- [ ] Refactoring (`refactor`): A code change that neither fixes a bug nor adds a feature, e.g Deleting unused or redundant code
- [ ] Documentation (`docs`): Documentation only changes
- [ ] Other (`style`, `build`, `ci`, `performance`): Formatting, linting or styling. Changes to the CI pipeline. Performance improvements. Changes that affect the build system.

### CODE REVIEW

File: .bumpversion-chart.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 8.3.0
+current_version = 9.0.0
```
Comment (deontpearson) [inline .bumpversion-chart.cfg:2]: I dont think this is a breaking change. Given that you are checking if the `podAnnotations` are defined and only then add them to the charts I would say this is a none breaking change, a patch or if you prefer a minor change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 - what changed
 
+## [9.0.0] - 2022-07-19
+
+### Added
+
+- Added the abliity to configure annotations applied to the pods of deployments and cronjobs
```
Comment (NJABULO-NDLOVU) [inline CHANGELOG.md:21]: typo

Comment (deontpearson) [inline CHANGELOG.md:21]:
```suggestion
- Added the abliity to configure annotations applied to the pods of deployments and cronjobs through the `podAnnotations` key in each application
```
or something along those lines

File: charts/aruba-uxi/values.example.yaml [hunk 3]
```diff
@@ -74,6 +74,10 @@ aruba-uxi:
         name: service-account-name
         ## Any annotations to add the service account that is created (optional, default: {})
         annotations: {}
+      ## Adds the specified annotations to the metadata of the pods
```
Comment (deontpearson) [inline charts/aruba-uxi/values.example.yaml:77]:
```suggestion
      ## Adds the specified annotations to the metadata of the pods (optional, default: {})
```

File: charts/aruba-uxi/values.example.yaml [hunk 4]
```diff
@@ -74,6 +74,10 @@ aruba-uxi:
...
+      podAnnotations:
+        example/annotation: 'foo'
+        example/annotation.for.spirit: 'bar'
```
Comment (deontpearson) [inline charts/aruba-uxi/values.example.yaml:80]:
```suggestion
      podAnnotations: {}
```
Do you think setting it to the default value looks better or conveys the way its used?

File: charts/aruba-uxi/values.example.yaml [hunk 5]
```diff
@@ -277,6 +281,10 @@ aruba-uxi:
         tag: 1.2.3
         ## The image pull policy to use (optional, default: IfNotPresent)
         pullPolicy: IfNotPresent
+      ## Adds the specified annotations to the metadata of the cronjob pod
+      podAnnotations:
+        example/annotation: 'foo'
+        example/annotation.for.spirit: 'bar'
```
Comment (deontpearson) [inline charts/aruba-uxi/values.example.yaml:287]:
```suggestion
      ## Adds the specified annotations to the metadata of the cronjob pod (optional, default: {})
      podAnnotations: {}
```
Same as above

File: charts/aruba-uxi/values.example.yaml (hunk 4 above)
Comment (MaraisKruger) [reply charts/aruba-uxi/values.example.yaml:80]: I like this better

**Review by brandon-piner-aruba — APPROVED** [review]

File: .vscode/settings.json [hunk 6]
```diff
@@ -0,0 +1,2 @@
+{
+}
```
Comment (deontpearson) [inline .vscode/settings.json:2]: 👀

Comment (deontpearson) [inline .vscode/settings.json:2]: 👀

File: CHANGELOG.md [hunk 7]
```diff
@@ -14,6 +14,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 - what changed
 
+## [9.0.0] - 2022-07-19
```
Comment (deontpearson) [inline CHANGELOG.md:17]:
```suggestion
## [8.4.0] - 2022-07-19
```

File: .bumpversion-chart.cfg [hunk 8]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 8.3.0
+current_version = 8.5.0
```
Comment (deontpearson) [inline .bumpversion-chart.cfg:2]: hmm someone was doing it manually

Comment (MaraisKruger) [reply .bumpversion-chart.cfg:2]: yeah

**Pull request merged**